ACCESSIBILITY.md

```diff
@@ -0,0 +1,91 @@
+- Design (ADDING AS PROXY - how do we encompass right-handed bias? Usability of websites AND of physical hardware like computer mice?)
+
+We recognise the complex and interlinking ways in which intersectionality affects identity, and thereby, access or inclusion within any community project like _The Turing Way_. So in this sense, our accessibility practices may overlap with practices that enable inclusion more broadly within any community.
+
+In developing this policy, we aim to draw from the [access-centered framework](https://accesscenteredmovement.com/what-access-centered-means/) in describing access as a constant process that changes in each space with each individual. This document is a part of a broader collection of documentation across accessibility, listed below:
+- [Accessibility policy](https://github.com/the-turing-way/the-turing-way/blob/main/ACCESSIBILITY.md): Our vision and guiding principles for how we practice accessibility.
+- Community Handbook Accessibility Guidelines: Practical advice to build accessibility into your contributions and activity within the community
+- Accessibility Guide: A broader discussion of accessibility and access-related practices that extend beyond The Turing Way
+
+Each resource (our Accessibility Policy - this document, Community Handbook Accessibility Guidelines, and our Accessibility Guide) play a different yet overlapping role in enabling, documenting, and sharing our evolving language around accessible and inclusive collaboration.
+
+This policy is a work in progress, as are the practices that we employ as a community. Much like how _The Turing Way_ itself is always being written and rewritten, our accessibility policies and practices are constantly being updated as broader awareness grows and changes in technology are made.
+
+## 1.1 Core Principles
+
+These core principles are at the heart of our accessibility policy, and they extend beyond the web accessibility of our open access guides to the broader practices that we use as a community, expanded upon in Section 2.0.
+
+This means that questions of access are tied to how people read the guides as well as their ability to participate in the project and the community. Participation in _The Turing Way_ itself can be defined broadly, but doing so often requires navigating a number of different platforms. In all contexts, we aim to facilitate access in a broad sense through:
+
+1. **Removing barriers**: We actively advocate for and employ tactics that seek to remove barriers for any current or future participants in our community. This may include people who historically have been limited or structurally excluded from participating through educational, geographical, linguistic, or technical barriers.
+2. **Sharing openly**: We aim to draft, write, review, and edit our open access guides using open source practices. We do this in order to enable people from a broad range of backgrounds to get involved in the project.
```

Comment:
Can we provide a template of some of the open source practices we might be following?

This is just to enable readers have an idea of the guidelines we might be following, as not everyone might know what these open source "practices" are.